image/templates/compute-sled/ramdisk-02-trim.json

```
@@ -133,6 +133,6 @@
             "extsrc": "gzonly.txt",
             "owner": "root", "group": "sys", "mode": "0644" },
 
-        { "t": "pack_tar", "name": "gimlet-ramdisk.tar" }
+        { "t": "pack_tar", "name": "compute-sled-ramdisk.tar" }
```

Can we just consistently use "sled" everywhere, rather than "compute-sled" (which is longer, and hyphenated) in some places and "sled" in others?

```suggestion
        { "t": "pack_tar", "name": "sled-ramdisk.tar" }
```